docs/content/docs/start/react-tutorial.md

```diff
@@ -0,0 +1,295 @@
+---
+title: 'Tutorial: Create a Fluid Framework application with React'
```

Comment:
We're going to use this for our react "recipe", so move this page to docs/content/docs/recipes/react.md

Reply:
Actually, I prefer this after looking at the staging site. Leave this and I'll follow up on what to do with recipes.